types/account.d.ts

```diff
@@ -129,10 +132,12 @@ export type ResetPasswordPayload = {
 };
 
 export type ChangePasswordPayload = {
```

Comment:
I want to avoid changing these Payload objects since they are right now cleeng-specific (see LoginArgs vs LoginPayload as an example.)